requirements.txt

```
@@ -4,5 +4,5 @@ pip>=21.0,<23.2
 ruff==0.0.292
 
 # https://github.com/home-assistant/core/issues/95192
-urllib3==1.26.16
+urllib3==2.0.7
```

```suggestion
urllib3>=1.26.5,<2
```
This would match the versions currently allowed in Home Assistant ([source](https://github.com/home-assistant/core/blob/d4c5a93b638356686d003321919a1639a4c6f4e6/script/gen_requirements_all.py#L47))

Hopefully this separate requirement can be dropped before too long when https://github.com/home-assistant/core/issues/95192 gets fixed.